pkg/devfile/adapters/common/command.go

```diff
@@ -84,7 +101,7 @@ func validateCommand(data data.DevfileData, command common.DevfileCommand) (err
 
 	isActionValid := false
 	for _, component := range components {
-		if command.Exec.Component == component.Container.Name && isComponentSupported(component) {
+		if command.Exec.Component == component.Container.Name {
```

Comment:
don't we need the isComponentSupported check? It seems to be modified below

Reply:
this is a redundant check, we are already doing the same check in  `GetSupportedComponents`

https://github.com/openshift/odo/blob/master/pkg/devfile/adapters/common/utils.go#L101